docs/docs/en/guide/datasource/kyuubi.md

```diff
@@ -0,0 +1,42 @@
+# KYUUBI
```

Comment:
you should also add new docs to https://github.com/apache/dolphinscheduler/blob/9df89cf0cf6eef7875e1fc476aab38e48bd8bb33/docs/configs/docsdev.js#L363-L365 to show in our website

Reply:
done